pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -236,6 +236,12 @@ if (( "${NIX_DEBUG:-0}" >= 1 )); then
     printf "  %q\n" ${extraAfter+"${extraAfter[@]}"} >&2
 fi
 
+# Uses mktemp so should be done before PATH is restored
+if (( "${NIX_CC_USE_RESPONSE_FILE:-@use_response_file_by_default@}" >= 1 )); then
+    responseFile=$(mktemp "${TMPDIR:-/tmp}/cc-params.XXXXXX")
+    trap 'rm -f -- "$responseFile"' EXIT
```

Comment:
As I can see `trap` will run `rm` via restored `PATH`. So this approach helps with `mktemp`, but not with `rm`.